src/core/ref-counted-cache.js

```diff
@@ -8,6 +8,15 @@ class RefCountedCache {
         this.cache = new Map();
     }
 
+    // destroy all stored objects
+    destroy() {
+        console.log("RefCountedCache destroy");
+        this.cache.forEach((refCount, lightmap) => {
```

Comment:
This is a general purpose utility class, right? Yet it now explicitly mentions `lightmap`s. Should we rename `lightmap` to `cacheItem` or something?

Reply:
thanks for paying attention, not sure how I missed it.